Dockerfile

```diff
@@ -11,7 +11,9 @@ RUN go mod download
 COPY . .
 
 FROM base AS builder
```

Comment:
Maybe once has to specify `--platform` here as well, though I cannot tell for sure.

I suggest trying without and adding if it fails